src/Adapter/Email/EmailConfigurationTester.php

```diff
@@ -90,8 +90,8 @@ public function testConfiguration(array $config)
         $result = Mail::sendMailTest(
             Tools::htmlentitiesUTF8($smtpChecked),
             Tools::htmlentitiesUTF8($config['smtp_server']),
-            Tools::htmlentitiesUTF8($content),
-            Tools::htmlentitiesUTF8($subject),
+            Tools::htmlentitiesDecodeUTF8($content),
```

Comment:
The decode of  content and subject like &lt; to < and not convert all applicable characters to HTML entities not causes a security problem?

Reply:
Variables $subject and $content are hardcoded in the previous lines. The user cannot modify them, so I don't see a security problem.
```
$content = $this->translator->trans(
            'This is a test message. Your server is now configured to send email.',
            [],
            'Admin.Advparameters.Feature'
        );
        $subject = $this->translator->trans('Test message -- Prestashop', [], 'Admin.Advparameters.Feature');
```

However ... I actually wonder why `htmlentitiesUTF8()` was used here 😄 . I do not understand what it the benefit. Maybe the solution is to simply remove all `htmlentitiesUTF8()` calls and NOT use `htmlentitiesDecodeUTF8()` ?
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Tools.php#L951

@edelmacias if you need to perform `htmlentitiesDecodeUTF8()` (which actually uses https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Tools.php#L951) it's because the $content contains HTML entities, right?

Reply:
@matks yes, that is correct. For english it works fine, but once you start using other languages, they break. As the translation are only managed by authenticated back office users, I don't see any issue with the changes for this PR